src/dispatch/cli.py

```diff
@@ -697,15 +697,20 @@ def run_server(log_level):
     """Runs a simple server for development."""
     # Uvicorn expects lowercase logging levels; the logging package expects upper.
     os.environ["LOG_LEVEL"] = log_level.upper()
-    if not config.STATIC_DIR:
+    if not os.path.isdir(config.STATIC_DIR):
         import atexit
         from subprocess import Popen
 
         # take our frontend vars and export them for the frontend to consume
         envvars = os.environ.copy()
         envvars.update({x: getattr(config, x) for x in dir(config) if x.startswith("VUE_APP_")})
 
-        p = Popen(["npm", "run", "serve"], cwd="src/dispatch/static/dispatch", env=envvars)
+        p = Popen(
+            ["npm", "run", "serve"],
+            cwd=os.path.join("src", "dispatch", "static", "dispatch"),
+            shell=True,
```

Comment:
Adding `shell=True` breaks my local dev env. It should be working without it in Windows according to the [Popen docs](https://docs.python.org/3/library/subprocess.html#popen-constructor):
> On Windows with shell=True, the COMSPEC environment variable specifies the default shell. The only time you need to specify shell=True on Windows is when the command you wish to execute is built into the shell (e.g. dir or copy). You do not need shell=True to run a batch file or console-based executable.

Also, there are [security considerations](https://docs.python.org/3/library/subprocess.html#security-considerations) that need to be taken into account when using `shell=True`.

Can you try without it and let me know if it works for you?
```suggestion
```